scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2111,44 @@ def visit_call(self, node):
                     break
 
 
+class StringConcatenationChecker(checkers.BaseChecker):
+    """Custom pylint checker which prohibits use of string concatenation and
+     promotes string interpolation instead.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'string-concatenation'
+    priority = -1
+    msgs = {
+        'C0034': (
+            'at string %s, use string interpolation (\'string1%%s\' %% string2)'
+            ' rather than string concatenation (\'string1\' + \'string2\').',
```

Comment:
Can we show a simpler, static error message like this - `Avoid using string concatenation and replace it with string interpolation instead.`? Looking at the test for multiple string concatenation, I thought we can avoid having long messages for such cases, as the simpler one still conveys the idea.

Reply:
If you want I can implement it, but [like you said before](https://github.com/oppia/oppia/pull/11910#discussion_r574524239), current message is clearly demonstrating what the user needs to do in order to fix the issue. 
Let me know your thoughts about it.

Reply:
@tanishq67 The current message meets the purpose, I just felt we can make it simpler.

Reply:
>Can we show a simpler, static error message like this - Avoid using string concatenation and replace it with string interpolation instead.? Looking at the test for multiple string concatenation, I thought we can avoid having long messages for such cases, as the simpler one still conveys the idea.

@sajalasati PTAL at the new message.

Reply:
@sajalasati let me know your thoughts so that I can implement it fast and complete the linter test

Reply:
@tanishq67 Though the error msg being printed wouldn't look that weird, and would have worked in a general sense for all cases, but I am fine with your suggestion too.

Reply:
@tanishq67 Is this comment resolved?

Reply:
@DubeySandeep Yes, it is resolved